templates/Keys.podspec.json

```diff
@@ -18,7 +18,10 @@
     "tag": "23"
   },
   "platforms": {
-    "ios": "8.0"
+    "ios": "8.0",
+    "osx": "10.10",
+    "watchos": "8.3",
```

Comment:
There is no watchos 8.3. Latest is 4.3.
Hope you can support watchOS from 4.0

Reply:
I'm working on one